test/utils/index.js

```diff
@@ -76,5 +76,23 @@ describe('our index utilities', () => {
       expect(splitIntoRows([9, 8, 7, 6, 5, 4, 3], 2)).to.deep.equal([[9, 8], [7, 6], [5, 4], [3]]);
     });
   });
+
+  describe('have a abbrNum utility that', () => {
+    it('returns abbreviated a rounded number based on (1000(n) & maximum trillion)', () => {
+      var sourceNumberSet = [123,1266,125468,77547959,388475766449,48242478968789,482424789687893242];
+      var formattedNumberSet = [123,'1.3k','125.47k','77.548m','388.476b','48.242t','482,424.79t'];
+      for (let i = 0; i < sourceNumberSet.length;i++) {
+        expect(abbrNum(sourceNumberSet[i],i)).to.be.equal(formattedNumberSet[i]);
+      }
+    });
+
+    it('returns no abbreviation if a negative number or 0', () => {
```

Comment:
I don't think this second test is necessary. It was a limitation in the utility function that it didn't abbreviate negative numbers. I've fixed that [here](https://github.com/OpenBazaar/openbazaar-desktop/pull/1119).

I don't think we need to test that it doesn't abbreviate on zero. What it's really doing is not abbreviating on < 1000, since the minimum abbreviate threshold is 1000.

Reply:
Thanks for the feedback!

I have removed the test for 0 as per your comments. Although, given your PR #1119 - you've added functionality to check negatives on line 63, therefore I feel its necessary to test for it. Also, there is a small bug in your code for numbers in range of -1 to -999 because you're using absolute numbers. See my comments in your pull request for patch.